frameworks/angular/container/src/app/view1/view1.component.ts

```diff
@@ -1,32 +1,53 @@
-import { fin } from "@openfin/core";
-import { Component } from '@angular/core';
 import "@finos/fdc3";
-import * as Notifications from "@openfin/workspace/notifications";
+import { CommonModule } from "@angular/common";
+import { ChangeDetectionStrategy, Component } from "@angular/core";
+import { fin } from "@openfin/core";
```

Comment:
One thing to be aware of. Try using types from openfin/core.  e.g. import type. Content running inside of an OpenFin container has the fin api injected. Importing the core module will result in a lot of mock js being added to your bundle so we suggest just importing the type for the validation.

Reply:
I usually leave that to eslint to do automatically, so I've [added](https://github.com/built-on-openfin/frontend-framework-starter/commit/e57e1dc5386b7d453218b805ffbe8ebd1c8fa643) the necessary config files: